source/how-tos/update-firmware.rst

```diff
@@ -23,32 +23,43 @@ Creating firmware update capsule image requires the following steps:
 
 #. Run capsule tool to generate capsule image
 
-  Capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow::
+  Capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow.
 
-    usage: GenCapsuleFirmware.py [-h] -b IMAGE [-t {b,c}] -k PRIVKEY -o NEWIMAGE
-                                 [-q]
+  Capsule tool is capable of incorporating multiple firmware images into single capsule binary. Each firmware is identified and included in the capsule image using a GUID.
+
+    usage: GenCapsuleFirmware.py [-h] -p BIOS <BIOS_IMAGE> -p <GUID> <FW IMAGE BINARY 1>.....-p <GUID> <FW IMAGE BINARY n> -k PRIVKEY -o NEWIMAGE [-q]
 
     optional arguments:
       -h, --help            show this help message and exit
-      -b IMAGE, --image IMAGE
-                            Raw image into firmware update capsule image
-      -t {b,c}, --type {b,c}
-                            Image type b:Bootloader c:CfgData
+      -p  <GUID> <Payload Image>, 
+                            Payload image that goes into firmware update capsule

```

Comment:
I think we need to talk about <GUID> here since it could be guid or a special known component name, such as BIOS, CFGD, CSME, etc.

Reply:
never mind, I saw the notes.  But better to state referring notes for details on  <GUID>

Reply:
Added reference line